frontend/public/kubevirt/components/vm.jsx

```diff
@@ -314,27 +314,47 @@ const mapDispatchToProps = () => ({
   watchK8sList: actions.watchK8sList
 });
 
-const ConnectedNewVMWizard = ({ activeNamespace, resources, flatten, onHide, createVm }) => {
-  const namespaces = flatten(resources);
-  const templates = _.get(resources, TemplateModel.kind, {}).data;
-  const networkConfigs = _.get(resources, NetworkAttachmentDefinitionModel.kind, {}).data;
-  const storageClasses = _.get(resources, StorageClassModel.kind, {}).data;
-  const persistentVolumeClaims = _.get(resources, PersistentVolumeClaimModel.kind, {}).data;
+const getLoadingResource = (resources, kind) => {
+  let data;
+  if (resources[kind]) {
+    data = resources[kind].loaded ? resources[kind].data : undefined;
+  } else {
+    // if model is unknown (CRD not created in opeshift, etc..), resources[kind] is undefined
+    data = [];
+  }
+  return {
+    data,
+    loadError: _.get(resources, kind, {}).loadError
+  };
+};
+
+const ConnectedNewVMWizard = ({ activeNamespace, resources, onHide, createVm }) => {
+  const namespaces = getLoadingResource(resources, NamespaceModel.kind);
+  const templates = getLoadingResource(resources, TemplateModel.kind);
+  const networkConfigs = getLoadingResource(resources, NetworkAttachmentDefinitionModel.kind);
+  const storageClasses = getLoadingResource(resources, StorageClassModel.kind);
+  const persistentVolumeClaims = getLoadingResource(resources, PersistentVolumeClaimModel.kind);
+
+  const error = Object.keys(resources).find(key => resources[key].loadError);
+  if (error) {
+    errorModal({error});
```

Comment:
I started using this as well
little enhancement: 

```jsx
const Errors = ({errors}) => (
  <ul>
    {errors.map((error, idx) => <li key={idx}>{error}</li>)}
  </ul>
);
errorModal({
  error: (<Errors errors={errors} />)
});
```

Reply:
@suomiy will you create a PR with just `Error` component ?